waterbox/bsnescore/bsnes/sfc/ppu-fast/ppu.cpp

```diff
@@ -52,13 +54,21 @@ PPU::PPU() {
   }
 
```

Comment:
How about allocating `lines` itself here as well? Considering `sizeof(Line) == 312`, a `lines = alloc_invisible<Line>(240);` should give another ~70KB boost.

Reply:
312 is the size after, and in my testing I did see a decent reduction in state size.

Reply:
Not sure of the consequences of putting a struct with functions into the invisible pool, paging @nattthebear here.

Reply:
The pointers are just as stable as any others, so the same downside as everything else:  If you're wrong about it, the core becomes nondeterministic, or maybe crashes if it expects something to be restored there that's not.